modules/ROOT/pages/migration/index.adoc

```diff
@@ -321,6 +321,11 @@ The following `@cypher` field types have been removed from the `sort` argument b
   - `@cypher` fields with non-sortable type like `Node`, `List` etc.
 
 
+=== Removed `connectComponentsPlanner` from `cypherQueryOptions`
+
+Following the deprecation of the `connectComponentsPlanner` https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/query-tuning/#cypher-connect-components-planner, 
+the field `connectComponentsPlanner` is no longer accepted in the Context and is removed from the `cypherQueryOptions` object.
```

Comment:
```suggestion
Following the deprecation of the link:https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/query-tuning/#cypher-connect-components-planner[`connectComponentsPlanner`], the field is no longer accepted in the context and is removed from the `cypherQueryOptions` object.
```

what does "context" refer to here?

Reply:
Is the GraphQL context passed in the GraphQL resolver